<launch>
  <param name="owd/calibration_file" type="string"
         value="pantilt_joint_calibrations" />
  
  <param name="owd/home_position" type="string"
         value="0, 0" />
  <param name="owd/max_cartesian_velocity" type="string"
         value="0.7" />
  <param name="owd/canbus_number" type="int" value="0" />
  <param name="owd/hand_type" type="string" value="none" />
  <param name="owd/forcetorque_sensor" type="bool" value="false" />
  <param name="owd/owd_plugins" type="string" value=""/>
  <param name="owd/hold_starting_position" type="bool" value="false" />
  <param name="owd/gravity_vector" type="string" value="0,0,-1" />
  <param name="owd/log_controller_data" type="bool" value="true"/>
  <param name="owd/log_canbus_data" type="bool" value="true"/>
  <node pkg="pantilt" type="owd" output="screen" respawn="false"
        cwd="node" name="owd">
  </node>
</launch>

